### ISSUE TITLE
DDS 8.8 Alpha Luminance - incorrect conversion

### ISSUE THREAD

**Issue by Trava715** [microsoft/DirectXTex#272]
When trying to convert textures saved as 8.8AL to other formats using texconv the channels are converted incorrectly.

Sample texture, L8A8
![image](https://user-images.githubusercontent.com/43389722/163383765-c2d5ea75-c459-418e-9c58-50eb43f1e822.png)

L8A8 to 8888RGBA
![image](https://user-images.githubusercontent.com/43389722/163383044-8d825a14-1bdc-4612-bca3-9fad68192572.png)

L8A8 to BC3_UNORM
![image](https://user-images.githubusercontent.com/43389722/163382957-dbdcc7b1-a22b-41d2-99af-700d23031c08.png)

L8A8 to BC7_UNORM (opened in Nvidia Texture Tools, channels look the same as when converting to BC3)
![image](https://user-images.githubusercontent.com/43389722/163383223-8dc606d0-f42a-44ab-b171-7c6b62db7e25.png)

Textures saved as 8 Luminance and 8 Alpha convert correctly.
Attaching test 88AL texture below
[88AL_sample.zip](https://github.com/microsoft/DirectXTex/files/8488783/88AL_sample.zip)

**Comment by walbourn** [comment]
**TL;DR**: Use the ``-xlum`` texconv switch.

The DirectXTex library 'default conversions' for legacy DDS formats matches those from ``DDSTextureLoader``. These mappings are based on 'no-conversion' behavior so that the data is directly loaded into memory.

In this case, this means ``D3DFMT_A8L8`` maps to the only DXGI format that exactly matches its bit and channel count ``DXGI_FORMAT_R8G8_UNORM``. Therefore ``L8`` channel is going to map to the RED channel your target format, and the ``A8`` is going to map to the GREEN channel in the target format. That is why you end up with a "Yellow" shape with an all opaque-alpha channel.

Semantically, these are quite different of course, so there are a number of other ways to map them. The ``-xlum`` switch in texconv (which enables the DDS loader flag ``DDS_FLAGS_EXPAND_LUMINANCE``) will instead map the ``L8`` channel to RED, GREEN, and BLUE and the ``A8`` channel will end up in the alpha channel of the target format.

> If you were trying to render classic ``D3DFMT_A8L8`` content directly loaded into memory, you'd have the shader do the conversion from ``DXGI_FORMAT_R8G8_UNORM``.

**Comment by Trava715** [comment]
I see, very sorry, not sure how I missed that in the documentation.

**Comment by walbourn** [comment]
Not a problem!

I add tables to these wiki pages to help clarify this behavior in the docs.

https://github.com/microsoft/DirectXTK/wiki/DDSTextureLoader#remarks

https://github.com/microsoft/DirectXTex/wiki/DDS-I-O-Functions#remarks
